docker-compose/docker-compose.yml

```
@@ -0,0 +1,413 @@
+      - consume
+      - transactions
+      - --group
+      - transactions-consumer
+      - -X user=superuser
+      - -X pass=secretpassword
+      - -X brokers=redpanda-0:9092
+    image: docker.redpanda.com/redpandadata/${REDPANDA_DOCKER_REPO:-redpanda}:${REDPANDA_VERSION:-latest}
+    networks:
+      - redpanda_network
+    depends_on:
+    - deploytransform
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)